package.json

```diff
@@ -26,5 +26,8 @@
     "@types/request-promise-native": "^1.0.17",
     "request-promise-native": "^1.0.8",
```

Comment:
I believe `request-promise-native` need to be in the `dependencies` section rather than the `dev dependencies`